ballerina/sql_client.bal

```diff
@@ -141,6 +159,46 @@ public client class SQLClient {
         }
     }
 
+        # Retrieves the values of the 'many' side of an association.
```

Comment:
```suggestion
    # Retrieves the values of the 'many' side of an association.
```

Reply:
Fixed in [3ac4df4](https://github.com/ballerina-platform/module-ballerina-persist/pull/155/commits/3ac4df4447e4c445dda487eb932865f44b7f670c)